src/test/java/CalcEstimateTestChrome.java

```diff
@@ -1,93 +1,99 @@
+import driver.DriverSingleton;
 import io.github.bonigarcia.wdm.WebDriverManager;
-import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.Platform;
-import org.openqa.selenium.WebDriver;
+import model.CommitedUsage;
+import org.openqa.selenium.*;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.remote.RemoteWebDriver;
 import org.openqa.selenium.support.PageFactory;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import page.CalcPage;
+import page.MainPage;
+import page.TempMailPage;
+import service.CommitedUsageCreator;
 
 
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.util.ArrayList;
 import java.util.concurrent.TimeUnit;
 
-@Test
-public class CalcEstimateTestChrome {
-    static WebDriver driver;
+
+public class CalcEstimateTestChrome extends CommonConditions {
+//    static WebDriver driver;
 
     @Test(groups = "parallel")
-    public static void main(String[] args) throws InterruptedException {
+    public void estimateTest() {
 //        WebDriverManager.chromedriver().setup();
 //        driver = new ChromeDriver();
+//        driver = DriverSingleton.getDriver();
 
         //Add new code for Selenium Grid
-        ChromeOptions options = new ChromeOptions();
-        try {
-            driver = new RemoteWebDriver(new 	URL("http://localhost:4444/wd/hub"), options);
-        } catch (MalformedURLException e) {
-            e.printStackTrace();
-        }
+//        ChromeOptions options = new ChromeOptions();
+//        try {
+//            driver = new RemoteWebDriver(new 	URL("http://localhost:4444/wd/hub"), options);
+//        } catch (MalformedURLException e) {
+//            e.printStackTrace();
+//        }
 
 
         driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
         driver.get("https://cloud.google.com/");
         driver.manage().window().maximize();
 
-        MainPage MainPg = PageFactory.initElements(driver, MainPage.class);
-        CalcPage CalcPg = PageFactory.initElements(driver, CalcPage.class);
-        TempMailPage TempPg = PageFactory.initElements(driver, TempMailPage.class);
-
-        MainPg.searchCloudCalc();
-        MainPg.calcClick();
-
-        CalcPg.CloudSite();
-        driver.switchTo().frame(0);
-        driver.switchTo().frame("myFrame");
-        CalcPg.NumberOfInstances();
-        CalcPg.OperationSystem();
-        CalcPg.MachineClass();
-        CalcPg.Series();
-        CalcPg.MachineType();
-        CalcPg.AddGPU();
-        CalcPg.NumberOfGPU();
-        CalcPg.GPUType();
-        CalcPg.LocalSSD();
-        CalcPg.DatacenterLocation();
-        CalcPg.CommitedUsage();
-        CalcPg.AddToEstimate();
-        CalcPg.EmailEstimateButton();
+        MainPage mainPg = PageFactory.initElements(driver, MainPage.class); //replace with constrator
```

Comment:
`PageFactory.initElements(driver, MainPage.class);` is already included in the MainPage constructor. Why not to use it instead?